resources/statistics.mjs

```diff
@@ -62,23 +62,19 @@ export function supportedConfidenceLevels() {
 export function confidenceIntervalDelta(confidenceLevel, numberOfSamples, sum, squareSum) {
     const probability = 1 - (1 - confidenceLevel) / 2;
     if (!(probability in tDistributionInverseCDF)) {
-        throw (
-            "We only support "
-            + supportedConfidenceLevels()
-                .map(function (level) {
-                    return level * 100 + "%";
-                })
-                .join(", ")
-            + " confidence intervals."
-        );
+        throw `We only support ${supportedConfidenceLevels()
```

Comment:
Can we call this function outside the template literal? This is hard to read.
Maybe something like:
```
const supportedIntervals = supportedConfidenceLevels().map(function (level) { return `${level * 100}%`; });
throw `We only support ${supportedIntervals.join(", ")} confidence intervals.`
```

Reply:
Agreed - updated the file. Let me know if that works.